src/index.ts

```diff
@@ -49,6 +49,10 @@ export {
 } from './support_code_library_builder/world'
 export { parallelCanAssignHelpers }
 
+export const HookParallelMode = {
```

Comment:
This could be a [TS enum](https://www.typescriptlang.org/docs/handbook/enums.html)